cmd2/parsing.py

```diff
@@ -309,7 +315,12 @@ def parse(self, rawinput: str) -> Statement:
         try:
             output_pos = tokens.index(constants.REDIRECTION_OUTPUT)
             output = constants.REDIRECTION_OUTPUT
-            output_to = ' '.join(tokens[output_pos+1:])
+
+            # Check if we are redirecting to a file
```

Comment:
I like the approach of doing the extra string manipulation within the parsing logic which you have taken here.